apps/typegpu-docs/src/content/examples/simple/increment-tgsl/index.ts

```diff
@@ -46,3 +24,11 @@ async function doIncrement() {
   root.compute(increment);
   return await counterBuffer.read();
 }
+
+/** @button "Increment" */
+export function click() {
+  (async () => {
+    const result = await doIncrement();
+    table.innerText = `${result.x} ${result.y}`;
+  })();
+}
```

Comment:
Are async click handlers not supported? If not, can we make them supported? 🤔 
```suggestion
/** @button "Increment" */
export async function click() {
  const result = await doIncrement();
  table.innerText = `${result.x} ${result.y}`;
}
```